src/app/app.component.ts

```diff
@@ -1,10 +1,29 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+
+import { AuthenticationService } from './shared/services/authentication.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'my-app';
+  showLoginModal?: boolean;
+  userName: string = '';
```

Comment:
I need field showLoginModal for communication between AppComponent and LoginComponent in order AppComponent  could close LoginComponent after click "Login". I think it's just temporary solution, because I just need routing here